src/compiler/types.ts

```diff
@@ -1693,6 +1693,8 @@ namespace ts {
           * @param path The path to test.
           */
         fileExists(path: string): boolean;
+
+        readFile(path: string): string;

```

Comment:
@alexeagle, @chuckjaz, @basarat, @adidahiya, @ivogabe, @jbrantly, and @chancancode this is a breaking change in the API, it adds a requirement for a new method `ParseConfigHost.readFile` to handle config file inheritance. We would like to merge this in soon. Please let us know if you have any reservations about this change.